app/controllers/concerns/cms/public_filter/layout.rb

```diff
@@ -10,177 +10,177 @@ module Cms::PublicFilter::Layout
   end
 
   private
-    def filters
-      @filters ||= begin
-        request.env["ss.filters"] ||= []
-      end
+  def filters
+    @filters ||= begin
+      request.env["ss.filters"] ||= []
     end
+  end
 
-    def find_part(path)
-      part = Cms::Part.site(@cur_site).filename(path).first
-      return unless part
-      @preview || part.public? ? part.becomes_with_route : nil
-    end
+  def find_part(path)
+    part = Cms::Part.site(@cur_site).filename(path).first
+    return unless part
+    @preview || part.public? ? part.becomes_with_route : nil
+  end
 
-    def render_part(part, opts = {})
-      return part.html if part.route == "cms/free"
+  def render_part(part, opts = {})
+    return part.html if part.route == "cms/free"
 
-      path = "/.s#{@cur_site.id}/parts/#{part.route}"
-      spec = recognize_agent path, method: "GET"
-      return unless spec
+    path = "/.s#{@cur_site.id}/parts/#{part.route}"
+    spec = recognize_agent path, method: "GET"
+    return unless spec
 
-      @cur_part = part
-      controller = part.route.sub(/\/.*/, "/agents/#{spec[:cell]}")
+    @cur_part = part
+    controller = part.route.sub(/\/.*/, "/agents/#{spec[:cell]}")
 
-      agent = new_agent controller
-      agent.controller.params.merge! spec
-      agent.controller.request = ActionDispatch::Request.new(request.env.merge("REQUEST_METHOD" => "GET"))
-      resp = agent.render spec[:action]
-      body = resp.body
+    agent = new_agent controller
+    agent.controller.params.merge! spec
+    agent.controller.request = ActionDispatch::Request.new(request.env.merge("REQUEST_METHOD" => "GET"))
+    resp = agent.render spec[:action]
+    body = resp.body
 
-      if body =~ /\#\{.*?parent_name\}/
-        parent = Cms::Node.site(@cur_site).filename(@cur_main_path.to_s.sub(/^\//, "").sub(/\/[\w\-\.]*?$/, "")).first
-        if parent
-          body.gsub!('#{parent_name}', ERB::Util.html_escape(parent.name))
-          body.gsub!('#{parent.parent_name}', ERB::Util.html_escape(parent.parent ? parent.parent.name : parent.name))
-        end
+    if body =~ /\#\{.*?parent_name\}/
+      parent = Cms::Node.site(@cur_site).filename(@cur_main_path.to_s.sub(/^\//, "").sub(/\/[\w\-\.]*?$/, "")).first
+      if parent
+        body.gsub!('#{parent_name}', ERB::Util.html_escape(parent.name))
+        body.gsub!('#{parent.parent_name}', ERB::Util.html_escape(parent.parent ? parent.parent.name : parent.name))
       end
-
-      @cur_part = nil
-      body
     end
 
-    def render_layout(layout)
-      @cur_layout = layout
-      @cur_item   = @cur_page || @cur_node
+    @cur_part = nil
+    body
+  end
 
-      @window_name = @cur_site.name
-      @window_name = "#{@cur_item.name} - #{@cur_site.name}" if @cur_item.filename != "index.html"
+  def render_layout(layout)
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for render_layout is too high. [79.76/68] [:ref](http://c2.com/cgi/wiki?AbcMetric)

<!-- Comment by [SideCI](https://sideci.com) -->